### DISCUSSION TITLE
Compiling Error VSCODE

### DISCUSSION THREAD

**Discussion by Gokuh1980** [Jyers/Marlin#766] [General Discussion]
Does anyone have the solution for this problem? I can´t find the solution.

![image](https://user-images.githubusercontent.com/5133989/117677305-b8410380-b184-11eb-9b38-c04abc4be757.png)

**Comment by Gokuh1980** [comment]
Sorry, i forgot to tell that Marlin (original branch) compiles without any errors...

**Comment by Cal-LF** [comment]
See #762.

**Reply by Gokuh1980** [reply]
Yes, i´ve tried the templates but the compiler gives me the same error... Any idea?

**Reply by Cal-LF** [reply]
I just tried the 3X3 man mesh and it compiled ok. I have the V427 board. Download the main github zip again and copy the template you use to the Marlin folder. Retry compile.

**Reply by Gokuh1980** [reply]
I did the same with 3X3 manual mesh but for 4.2.2 and get the same error...

**Reply by Cal-LF** [reply]
I just compiled with the V422 board and it is ok. 
If Jacob M checks this post and the error you have, i'm sure he can help.
I have had this same error in the past also.

**Reply by Gokuh1980** [reply]
i don´t know, i´ve tried with two different installations, several things, etc, it´s a problem with the display i think...

**Reply by Tuncay-Ayhan** [reply]
> i don´t know, i´ve tried with two different installations, several things, etc, it´s a problem with the display i think...

I can compile every version of the .BIN files fine with or without VSCode. 

Maybe you can read the great tips people gave to Jacob(Jyers), when he had problems with VSCode. #723

**Comment by Jyers** [comment]
Based on the error, there is an issue with your files. Try a fresh download/clone. Also make sure you are on the correct github branch.

**Reply by Gokuh1980** [reply]
Sorry for the delay, i did what you told me and the same result... I am in the correct branch i think, can you send me a link just to be sure? Thanks

**Reply by tome9111991** [reply]
lcd/extui/lib/ not extis in  latest update

try 
https://github.com/Jyers/Marlin/archive/refs/heads/Ender_3_V2_Extensible_UI.zip

**Reply by Gokuh1980** [reply]
Nice, it compiles without errors, i´ve tested Template configuration and, Jyers configuration.
Another thing i did is put the directory in the root of my HDD.


THANK YOU ALL!!!

**Reply by Gokuh1980** [reply]
oh, just one more answer, that file you point me in the link, what is?

**Reply by tome9111991** [reply]
Is latest firmware from jyers.
I have no problem to compile

**Reply by Gokuh1980** [reply]
Excuse me the noob question, every time jyers publish one release i should redownload from the link you sended?

https://github.com/Jyers/Marlin/archive/refs/heads/Ender_3_V2_Extensible_UI.zip

**Reply by tome9111991** [reply]
if you have still compiling erros you can try this link.
but it the  same version on mainpage
